humanoid_league_game_controller/config/game_controller.yaml

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
rename this file to game_controller.yaml.example so it is not used by mistake

Reply:
I split the settings that should be changed and the persistent settings in two files because one of them is replaced by the robot_compile script.